app/controllers/posts_controller.rb

```diff
@@ -8,4 +8,28 @@ def show
     @user = User.find(params[:user_id])
     @post = Post.find(params[:id])
   end
+
+  def new
+    @current = current_user
+  end
+
+  def create
+    new_post = current_user.posts.build(post_params)
```

Comment:
The user is encountered with an error if he tries to click on a post in order to add a comment. Please see the error and make sure the current user is able to add comments to a post.
![new_post](https://user-images.githubusercontent.com/35504149/155699506-5e49c16c-2d25-4a28-95fb-23f0ea79f0ff.png)

Reply:
resolved